packages/app/src/cli/services/generate.ts

```diff
@@ -59,7 +60,9 @@ async function generate(options: GenerateOptions) {
       )
     }
   } else {
-    specifications = specifications.filter((spec) => app.extensionsForType(spec).length < spec.registrationLimit)
+    specifications = specifications
```

Comment:
@isaacroldan  we have to manually filter this out for the generate command for now since we don't yet have a way to enable multiple templates for a single specification

Reply:
Makes sense!
Did you see my proposal for a new template interface in the CLI? https://github.com/Shopify/internal-cli-foundations/discussions/518
If we agree on this, we might be able to work on it soon